picard/metadata.py

```diff
@@ -174,41 +174,40 @@ def compare_to_release_parts(self, release, weights):
 
         if "releasetype" in weights:
             type_scores = dict(config.setting["release_type_scores"])
-            if 'release_group' in release.children and 'type' in release.release_group[0].attribs:
-                release_type = release.release_group[0].type
+            if 'release-group' in release and 'primary-type' in release['release-group']:
+                release_type = release['release-group']['primary-type']
                 score = type_scores.get(release_type, type_scores.get('Other', 0.5))
             else:
                 score = 0.0
             parts.append((score, weights["releasetype"]))
 
-        rg = QObject.tagger.get_release_group_by_id(release.release_group[0].id)
-        if release.id in rg.loaded_albums:
+        rg = QObject.tagger.get_release_group_by_id(release['release-group']['id'])
+        if release['id'] in rg.loaded_albums:
             parts.append((1.0, 6))
 
         return parts
 
     def compare_to_track(self, track, weights):
         parts = []
-
         if 'title' in self:
             a = self['title']
-            b = track.title[0].text
+            b = track['title']
```

Comment:
I just got a TypeError here:

```
Traceback (most recent call last):
  File "./picard/webservice/__init__.py", line 266, in _process_reply
    self._handle_reply(reply, request, handler, parse_response_type, refresh)
  File "./picard/webservice/__init__.py", line 255, in _handle_reply
    handler(document, reply, error)
  File "./picard/acoustid/__init__.py", line 93, in _on_lookup_finished
    next_func(doc, http, error)
  File "./picard/file.py", line 599, in _lookup_finished
    reverse=True, key=itemgetter(0))[0]
  File "./picard/file.py", line 598, in <genexpr>
    track, self.comparison_weights) for track in tracks),
  File "./picard/metadata.py", line 194, in compare_to_track
    b = track['title']
TypeError: 'NoneType' object is not subscriptable
```

I was trying to load https://musicbrainz.org/release/772ae1d4-4fa8-4031-83b0-37c8f38df166.